features/provider/active_docs.feature

```diff
@@ -0,0 +1,36 @@
+Feature: ActiveDocs pages
+  As a provider
+  I want to manage my ActiveDocs
+
+  Background:
+    Given a provider is logged in
+      And the provider has 1 active doc
+
+  Scenario: Index renders with the proper elements
+    When I go to the provider active docs page
+    Then I should see "ActiveDocs" within the main title
+     And the table should contain the API
+
+  Scenario: Preview renders with the proper elements
+    When I go to the preview active docs page
+    Then I should see "ActiveDocs" within the main title
```

Comment:
Is this step necessary for every scenario?

Reply:
It is one of the things that we were testing through the xpath before 😕 So I'd say yes 🤔

Reply:
Well it is not really informative in cucumber. 
I'd either remove it as it is a detail not important for the reader/feature.